tests/utils.py

```diff
@@ -647,6 +647,9 @@ def wrapper(*args, **kwargs):
     'frozenset': [
             'frozenset()',
             'frozenset({1, 2.3456, "another"})',
+            'frozenset([1, 2.3456, "another"])',
+            'frozenset((1, 2.3456, "another"))',
+            'frozenset(iter([1, 2.3456, "another"]))',
```

Comment:
It looks like you're trying to test the frozenset constructor here. This is not the best place to do it, as the `SAMPLE_DATA` is meant to provide a variety of values, not a bunch of the same values.

Instead, these tests should go into `tests/datatypes/test_frozenset.py`, under the 'FrozensetTests' test case.